addon/components/layers-dialogs/edit-modes/base.js

```diff
@@ -29,7 +29,8 @@ export default Ember.Component.extend({
     @method bindProperties
   */
   bindProperties() {
-    let parent = this.get('parentView.parentView');
+    // Instance of flexberry-edit-layermap component.
+    let parent = this.get('parentView.parentView.parentView');
```

Comment:
I suppose it will be better to search for parent in while cycle until `parentView instance of FlexberryEditLayermap` contition became true.

Reply:
fixed